src/Microsoft.Performance.SDK/Processing/ProcessingSourceInfo.cs

```diff
@@ -41,6 +68,29 @@ public sealed class ProcessingSourceInfo
     /// </summary>
     public sealed class ContactInfo
     {
+        /// <summary>
+        ///    Initializes a new instance of the <see cref="ContactInfo"/> class with the specified parameters.
+        /// </summary>
+        [JsonConstructor]
+        public ContactInfo(
+            string name,
+            string address,
+            string[] emailAddresses,
+            string[] phoneNumbers)
+        {
+            this.Name = name;
+            this.Address = address;
+            this.EmailAddresses = emailAddresses;
+            this.PhoneNumbers = phoneNumbers;
+        }
+
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="ContactInfo"/> class.
+        /// </summary>
+        public ContactInfo()
```

Comment:
What's the reason for these empty ctors?

Reply:
These are currently used by `ProcessingSource` in `GetAboutInfo()`. Would be a breaking change to remove these.